Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM emscripten/emsdk:3.1.32
+WORKDIR /app
+
+COPY ./package.json .
+COPY ./vite.config.ts .
```

Comment:
@AntonOstman, are these copy instructions necessary if we copy everything att line 9?